homeassistant/components/vera.py

```diff
@@ -164,7 +164,7 @@ def device_state_attributes(self):
         attr = {}
 
         if self.vera_device.has_battery:
-            attr[ATTR_BATTERY_LEVEL] = self.vera_device.battery_level + '%'
+            attr[ATTR_BATTERY_LEVEL] = self.vera_device.battery_level
```

Comment:
yes (or int if it's just a number?). And catch the `TypeError` in case the conversion fails. When failed, just not add that attribute (that way the data in that field stays of a consistent type)